src/Compilers/CSharp/Test/Symbol/Compilation/GetSemanticInfoTests.cs

```diff
@@ -6003,5 +6003,29 @@ class B<T, U, U>
             symbol = model.GetDeclaredSymbol(typeParameters[typeParameters.Length - 2]);
             Assert.True(symbol.IsReferenceType);
         }
+
+        [Fact]
+        [WorkItem(25262, "https://github.com/dotnet/roslyn/issues/25262")]
+        public void IncompleteInvocationSpeculativeSymbolInfo()
+        {
+            var source = @"
+class Foo
+{
+    void Bar(string s)
+    {
+        s?.ToString();
+    }
+}";
+
+            var comp = CreateCompilation(source);
+            var tree = comp.SyntaxTrees.Single();
+            var model = comp.GetSemanticModel(tree);
+
+            var invocation = tree.GetRoot().DescendantNodes().OfType<InvocationExpressionSyntax>().First();
+            // disconnect invocation from its Parent
+            var newInvocation = invocation.WithoutTrivia();
```

Comment:
>var newInvocation = invocation.WithoutTrivia(); [](start = 12, length = 47)

Please add a test where a complete conditional invocation node is constructed, but only the invocation child is passed to the GetSpeculativeSymbolInfo API. I would expect it to fail.